integration-tests/tests/run-workflow-on-snapshot.js

```diff
@@ -0,0 +1,83 @@
+const _ = require('lodash/fp')
+const pRetry = require('p-retry')
+const { withWorkspace } = require('../utils/integration-helpers')
+const { click, clickable, delay, dismissNotifications, findElement, findText, fillIn, input, select, signIntoTerra, waitForNoSpinners, findInGrid, navChild } = require('../utils/integration-utils')
+const { withUserToken } = require('../utils/terra-sa-utils')
+
+
+const snapshotName = 'testsnapshot'
+const findWorkflowButton = clickable({ textContains: 'Find a Workflow' })
+
+const testRunWorkflowOnSnapshotFn = _.flow(
+  withWorkspace,
+  withUserToken
+)(async ({ dataRepoUrl, environment, page, testUrl, snapshotColumnName, snapshotId, snapshotTableName, token, workflowName, workspaceName }) => {
+  if ( environment == "perf") {
+    // do nothing
+  } else {
+    // IMPORT SNAPSHOT
+    await page.goto(testUrl + "/#import-data?url=" + dataRepoUrl + "&snapshotId=" + snapshotId + "&snapshotName=" + snapshotName + "&format=snapshot")
```

Comment:
Instead of concatenating strings, please convert this to a template string. (Here and for other string concatenation instances in this PR)

If you hit option+return on this string, IntelliJ should offer to do it for you.